dags/app/cloudflare_bot/dag.py

```diff
@@ -0,0 +1,74 @@
+"""
+Create bot network traffic
+"""
+import logging
+from datetime import datetime, timedelta
+
+import requests
+from airflow.sdk import Variable, dag, task
+
+DEFAULT_ARGS = {
+    "owner": "Henry Lee",
+    "depends_on_past": False,
+    "start_date": datetime(2025, 1, 1),
+    "retries": 2,
+    "retry_delay": timedelta(minutes=5),
+}
+
+logger = logging.getLogger(__name__)
+
+
+@dag(
+    default_args=DEFAULT_ARGS,
+    schedule="@hourly",
+    max_active_runs=1,
+    catchup=False,
+)
+def PYCONTW_ETL_BOT_v1():
+
+    @task
+    def GET_TOP_WEBSITES() -> list[str]:
+        """Call Cloudflare Radar and return a list of the top-100 domains.
+
+        Docs: https://developers.cloudflare.com/api/resources/radar/subresources/ranking/methods/top/
+        """
+        token = Variable.get("CLOUDFLARE_RADAR_API_TOKEN")
+
+        url = "https://api.cloudflare.com/client/v4/radar/ranking/top"
```

Comment:
I would suggest we use https://airflow.apache.org/docs/apache-airflow-providers-http/stable/operators.html insetad